.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,25 @@
+Thank you for submitting the pull request to the Apache Ignite.
+
+In order to streamline the review of the contribution 
+we ask you to ensure the following steps have been taken:
+
+### The Contribution Checklist
+- [ ] The JIRA ticket has been associated with the pull request
```

Comment:
Associated how?

Reply:
Fixed. 
The weblink must exist on the PR in the JIRA ticket.